commit 8669cbc5e651bf4effa20e8c244a5a7d67da6fe9
Author: Thomas Hellstrom <thomas@tungstengraphics.com>
Date:   Mon Aug 7 22:22:10 2006 +1000

    drm: move drm authentication to new generic hash table.
    
    Fix drm_remove_magic potential memory leak / corruption. Move drm
    authentication token hashing to new generic hash table implementation.
    
    Signed-off-by: Dave Airlie <airlied@linux.ie>

diff --git a/drivers/char/drm/drm_hashtab.h b/drivers/char/drm/drm_hashtab.h
index 40afec05bff8..9e19ef1814ba 100644
--- a/drivers/char/drm/drm_hashtab.h
+++ b/drivers/char/drm/drm_hashtab.h
@@ -35,7 +35,7 @@
 #ifndef DRM_HASHTAB_H
 #define DRM_HASHTAB_H
 
-#define drm_hash_entry(_ptr, _type, _member) container_of(_ptr, _type, _member)
+#define drm_hash_entry(_ptr, _type, _member) list_entry(_ptr, _type, _member)
 
 typedef struct drm_hash_item{
 	struct hlist_node head;